posts/app.py

```diff
@@ -0,0 +1,66 @@
+"""
+posts/app.py
+Authors: mukobi
+Main flask app for posts
+    - Serve stored media
+    - Upload new posts to database
+
+
+Copyright 2019 The Knative Authors
```

Comment:
This can be a comment rather than a docstring.

Reply:
According to [PEP 257](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring), "A docstring is a string literal that occurs as the first statement in a module ... All modules should normally have docstrings." This is passed to the __doc__ special attribute when importing.

Reply:
Sorry, I meant the copyright statement can be a comment, rather than being part of the docstring. So:

```python
"""Main application for posts:
  - Serve stored media
  - Upload new posts to database
"""
# Authors: ....
# Copyright 2019 ....
# ... and more copyright

Reply:
Done https://github.com/knative-portability/large-events/pull/3/commits/5a4b66fa2055c5676726c33fd69153a843a8db0d https://github.com/knative-portability/large-events/pull/3/commits/530628bea576ab2867070c070579dd861eb250e6